fiftyone/constants.py

```diff
@@ -41,7 +41,7 @@
 # This setting may be ``None`` if this client has no compatibility with other
 # versions
 #
-COMPATIBLE_VERSIONS = ">=0.19,<0.22"
+COMPATIBLE_VERSIONS = ">=0.19,<0.23"
```

Comment:
Looks like this was not bumped for `fiftyone=0.22`, which could cause issues for users that are trying to connect with different client versions to the same database (usually in Teams context).

Reply:
Seems to work fine on the lower bound:
```
❯ fiftyone --version
Traceback (most recent call last):
  File "/home/topher/.virtualenvs/fiftyone/bin/fiftyone", line 5, in <module>
    from fiftyone.core.cli import main
  File "/home/topher/.virtualenvs/fiftyone/lib/python3.10/site-packages/fiftyone/__init__.py", line 34, in <module>
    _fom.migrate_database_if_necessary()
  File "/home/topher/.virtualenvs/fiftyone/lib/python3.10/site-packages/fiftyone/migrations/runner.py", line 147, in migrate_database_if_necessary
    runner = MigrationRunner(head, destination)
  File "/home/topher/.virtualenvs/fiftyone/lib/python3.10/site-packages/fiftyone/migrations/runner.py", line 300, in __init__
    raise EnvironmentError(
OSError: You must have fiftyone>=0.22.1 installed in order to migrate from v0.22.1 to v0.18.0, but you are currently running fiftyone==0.18.0.

See https://voxel51.com/docs/fiftyone/getting_started/install.html#downgrading-fiftyone for information about downgrading FiftyOne.
```

Reply:
The problem is more specific: setting `COMPATIBLE_VERSIONS = ">=0.19,<0.23"` would allow for a user running `fiftyone==0.22.2` to connect to a database that has been upgraded to a hypothetical future version of say 0.22.51.

For example, since we didn't bump `COMPATIBLE_VERSIONS` for 0.22.0, a user running `fiftyone==0.22.0` would not be able to connect to a database that's been upgraded to 0.22.X.

Reply:
Perhaps db <> client version mismatches are only ever allowed/relevant when `FIFTYONE_DATABASE_ADMIN=false`. I forget the details.